spec/swagger.yaml

```diff
@@ -678,3 +955,49 @@ definitions:
         default: true
       avatar:
         type: integer
+  Barcode:
+    type: object
+    required:
+      - id
+      - barcode
+      - type
+      - linked
+      - created_at
+      - updated_at
+    properties:
+      id:
```

Comment:
Do barcodes need to have an `id`? The barcode content should be unique, anyway.
(If they don't have one, we won't be able to update a barcode later on. But this might not be needed.)

Reply:
The barcodes have to be unique, otherwise you won't be able to tell them apart. I tried to maintain an unique way to form the api endpoints. Therefore I decided to assign an id to them.